src/org/skypro/skyshop/search/BestResultNotFound.java

```diff
@@ -0,0 +1,15 @@
+package org.skypro.skyshop.search;
+
+public class BestResultNotFound extends Exception{
```

Comment:
названия классов исключений должны оканчиваться на Exception
```suggestion
public class BestResultNotFoundException extends Exception{
```